CMakeLists.txt

```diff
@@ -94,6 +101,14 @@ if(NOT CC_OPTIMIZE)
 endif()
 
 if (MSVC)
+  if (MSVC)
+    check_cxx_compiler_flag(-sdl HAS_SDL)
+  endif()
```

Comment:
can this be moved closer to where HAS_SDL is used?

Reply:
No, a lot of SDL error actually happened on 3rd libraries and even some code in ort-extensions itself. So right now, I can only turn it on lib by lib, and there would be multiple places to add SDL flag.